### DISCUSSION TITLE
ldap user is not support

### DISCUSSION THREAD

**Discussion by rulerp** [apache/dolphinscheduler#9414] [Q&A]
[ERROR] 2022-04-09 18:17:46.934 org.apache.dolphinscheduler.server.worker.runner.TaskExecuteThread:[133] - tenantCode: u_test3 does not exist

private static List<String> getUserListFromLinux() throws IOException {
        List<String> userList = new ArrayList<>();

        try (BufferedReader bufferedReader = new BufferedReader(
                new InputStreamReader(new FileInputStream("/etc/passwd")))) {
            String line;

            while ((line = bufferedReader.readLine()) != null) {
                if (line.contains(":")) {
                    String[] userInfo = line.split(":");
                    userList.add(userInfo[0]);
                }
            }
        }

        return userList;
    }

--------

**Comment by rulerp** [comment]
ldap user is not wrriten into /etc/passwd file

**Reply by HwiLu** [reply]
how to solve it?
